### PR TITLE
Added dev command.

### PR DESCRIPTION
**Pull request by nicolad**

### CODE REVIEW

**Comment by DirtyF** [comment]
@nicolad thanks, can you explain you're solving here?

File: package.json [hunk 1]
```diff
@@ -60,6 +60,7 @@
   "scripts": {
     "build": "gatsby build",
     "develop": "gatsby develop",
+    "dev": "rm -rf ./cache && gatsby develop",
```
Comment (DirtyF) [inline package.json]:
```suggestion
    "clean": "rm -rf ./cache",
```

Let's add a `clean` script and call it in another command.

**Comment by nicolad** [comment]
> @nicolad thanks, can you explain you're solving here?

sure, `dev` command is something I noticed in a few gatsby projects and its main purpose is the convenience to type less

**Comment by spbyrne** [comment]
Nice, thanks @nicolad! Seems useful.

**Pull request merged**